docker-compose.yml

```diff
@@ -80,7 +80,10 @@ services:
     volumes:
       - ctfeConfig:/etc/config/:rw
   ct_server:
-    image: ghcr.io/sigstore/scaffolding/ct_server:v0.7.23@sha256:47170e948ac210422de689168a2422c7223f5c45121d910b9168be41deef66d2
+    build:
+      context: .
+      dockerfile: Dockerfile.ct_server
+    user: root
```

Comment:
Do we have to specify the user? The default is root if unset.

Reply:
No need. removed.